client/pom.xml

```diff
@@ -29,5 +29,11 @@
       <groupId>io.dropwizard</groupId>
       <artifactId>dropwizard-client</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
There were no unit tests in the client sub-project, so I had to add a JUnit dependency in order to add them.